target/linux/ipq806x/image/Makefile

```diff
@@ -316,4 +316,21 @@ define Device/zyxel_nbg6817
 endef
 TARGET_DEVICES += zyxel_nbg6817
 
+define Device/askey_rt4230w
```

Comment:
What happened to the revision here?

Reply:
I want to keep it clean so I ended up putting the revision in the device name. Also I tried adding _rev6 to that line, but then make started looking for a dts file named qcom-ipq8065-rev6.dts when trying to build.

Reply:
Excuse me, where can I download initramfs image?

Reply:
Image and instructions for installing to nand here: https://github.com/lmore377/openwrt-rt4230w

Reply:
> Excuse me, where can I download initramfs image?
@lxl750629
Image and instructions for installing to nand here: https://github.com/lmore377/openwrt-rt4230w
Discussion is here: https://forum.openwrt.org/t/askey-rac2v1k-support/15830